scenes/MosaikHeer.scn

```diff
@@ -12,15 +12,15 @@
 /config/talk/A  +2.3 OFF ON %000000001111111111
 /config/talk/B -11.0 OFF ON %010000000000000000
 /config/osc -41.0 100.2 1k00 F1 PINK 18
-/config/routing REC
 /config/routing/IN A1-8 A9-16 A17-24 A25-32 AUX1-4
 /config/routing/AES50A OUT1-8 OUT9-16 OUT1-8 OUT1-8 OUT1-8 OUT1-8
 /config/routing/AES50B OUT9-16 OUT9-16 OUT9-16 OUT9-16 OUT9-16 OUT9-16
 /config/routing/CARD OUT1-8 OUT9-16 A17-24 A25-32
 /config/routing/OUT OUT1-4 AUX/CR OUT9-12 OUT13-16
 /config/routing/PLAY AN1-8 AN1-8 AN1-8 AN1-8 AUX1-4
+/config/routing REC
 /config/userctrl/A WH
-/config/userctrl/A/enc "X001" "X101" "X209" "X202"
+/config/userctrl/A/enc "X001" "X101" "X203" "X205"
```

Comment:
Can you explain, what has changed here?

Reply:
This line describes the assignment of the rotary encoders in the ASSIGN section to the FX parameters. Since the delay (in slot 3: "X2") was changed the parameter number changed, too (from parameter 09 to 03 and from 02 to 05 / Feedback and balance between delay/chorus).

(see page 104 here: https://wiki.munichmakerlab.de/images/1/17/UNOFFICIAL_X32_OSC_REMOTE_PROTOCOL_%281%29.pdf)

Reply:
That makes sense. Thank you